csharp/ql/lib/semmle/code/csharp/dataflow/NegativeSummary.qll

```diff
@@ -0,0 +1,9 @@
+/** Provides modules for importing negative summaries. */
+
+/**
+ * A module importing the frameworks that provide external flow data,
+ * ensuring that they are visible to the taint tracking / data flow library.
+ */
+private module Frameworks {
```

Comment:
I think I would move this file into `internal`.

Reply:
Good idea!